wayland-backend/src/client_api.rs

```diff
@@ -123,7 +145,10 @@ impl Backend {
     /// Try to initialize a Wayland backend on the provided unix stream
     ///
     /// The provided stream should correspond to an already established unix connection with
-    /// the Wayland server. On this rust backend, this method never fails.
+    /// the Wayland server.
+    ///
+    /// This method can only fail on the `sys` backend if the `dlopen` cargo feature was enabled
```

Comment:
sys backend is an internal thing, probably should mention the feature.

Reply:
`wayland-backend` exposes both `rs` and `sys` modules in the crate root, and they are referred as such in the crate-level documentation, so imo that does not qualify as "an internal thing".